packages/react/src/themes/teams/components/Animation/animationStyles.ts

```diff
@@ -1,5 +1,23 @@
+import { AnimationProp } from '../../../types'
+import createAnimationStyles from '../../../../lib/createAnimationStyles'
+
 export default {
   root: () => ({
     display: 'inline-block',
   }),
+  children: ({ props, variables, theme }) => {
```

Comment:
It's strange to have `children` slot, but I don't have better idea

Reply:
I was struggling a bit with this one as well, but this is the most correct..